main.tf

```diff
@@ -1 +1,21 @@
-# A file for main terraform configuration
+resource "aws_instance" "windows_ec2" {
```

Comment:
Can you think of an easy way to encapsulate all three resources?
Maybve something that already exists?

Reply:
I commited a fix for that - using existing aws ec2 module from terraform registry.